### PR TITLE
Javadoc markup fixes

### PR DESCRIPTION
**Pull request by vojtechtoman**
Fixed the many markup-related errors (character escaping,
invalid markup, etc.). Most of the actual javadoc issues like
missing javadoc, broken references, etc. still present
and will need to be addressed one by one.

Note to reviewers - you can auto-expand files changed by using this script in your browser: https://github.com/hazelcast/hazelcast/pull/14337#issuecomment-451113431

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheReplicationOperation.java [hunk 1]
```diff
@@ -149,6 +148,11 @@ public String getServiceName() {
         return ICacheService.SERVICE_NAME;
     }
 
+    @Override
```
Comment (mmedenjak) [inline hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheReplicationOperation.java]: Minor: is this needed?

File: hazelcast/src/test/java/com/hazelcast/config/InvalidConfigurationTest.java [hunk 2]
```diff
@@ -474,30 +474,30 @@ private static String getDraftXml() {
                 + "</map>\n"
 
                 + "<cache name=\"default\">\n"
-                + "<key-type class-name=\"${cache-key-type-class-name}\"/>\n"
-                + "<value-type class-name=\"${cache-value-type-class-name}\"/>\n"
+                + "<key-type class-name=\"${cache-key-type-class-name}\">\n"
```
Comment (mmedenjak) [inline hazelcast/src/test/java/com/hazelcast/config/InvalidConfigurationTest.java]: Are these intentional? Same in other tests.

File: hazelcast/src/main/java/com/hazelcast/config/MemberDomConfigProcessor.java [hunk 3]
```diff
@@ -1664,14 +1664,14 @@ private NearCacheConfig handleNearCacheConfig(Node node) {
             String value = getTextContent(child).trim();
             if ("max-size".equals(nodeName)) {
                 nearCacheConfig.setMaxSize(Integer.parseInt(value));
-                LOGGER.warning("The element <max-size/> for <near-cache/> is deprecated, please use <eviction/> instead!");
+                LOGGER.warning("The element <max-size> for <near-cache> is deprecated, please use <eviction> instead!");
```
Comment (mmedenjak) [inline hazelcast/src/main/java/com/hazelcast/config/MemberDomConfigProcessor.java]: Are these intentional?

**Comment by mmedenjak** [comment]
Note to reviewers - you can auto-expand files changed by using this script in your browser: https://github.com/hazelcast/hazelcast/pull/14337#issuecomment-451113431

File: hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheReplicationOperation.java (hunk 1 above)
Comment (vojtechtoman) [reply hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheReplicationOperation.java]: No, I wonder how this got there. There are actually more places where I made changes to the code inadvertently - I will fix those.

File: hazelcast/src/test/java/com/hazelcast/config/InvalidConfigurationTest.java (hunk 2 above)
Comment (vojtechtoman) [reply hazelcast/src/test/java/com/hazelcast/config/InvalidConfigurationTest.java]: Definitely not intentional. Should be fixed in the latest commit.

File: hazelcast/src/main/java/com/hazelcast/config/MemberDomConfigProcessor.java (hunk 3 above)
Comment (vojtechtoman) [reply hazelcast/src/main/java/com/hazelcast/config/MemberDomConfigProcessor.java]: Definitely not intentional. Should be fixed in the latest commit.

**Comment by vojtechtoman** [comment]
run-lab-run

**Comment by pveentjer** [comment]
@vojtechtoman could you rebase so we can merge this pr?

**Comment by vojtechtoman** [comment]
run-lab-run

**Comment by vojtechtoman** [comment]
run-lab-run

**Comment by mmedenjak** [comment]
@vojtechtoman can you rebase and merge?

**Comment by vojtechtoman** [comment]
run-lab-run

**Pull request merged**